contrib/check-release.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source "${SCRIPT_DIR}/functions-sign.sh"
+
+declare -A SIGKEYS
+
+SIGKEYS["hexa-"]="c8e33aa86b1d3ad894d744d76232fa6325efb63672c3b972bb91f5197e2a96f9"
+SIGKEYS["braack"]="52b8c8de3985035ffcb0246e537396906357d8d244930947e4bb2c3da370fff7"
+SIGKEYS["andi-"]="73bea808dd08c77b4c68c80e9ebe10f5690459b77a4ad0e5074a4583e5775cbc"
+SIGKEYS["fluxx"]="af9a8c08f975d54c9015d015668d3a084e61af43180cbe23def3f79c6e80b32c"
+SIGKEYS["blocktrron"]="910ddca3b0561bebcb112ea20b714114fe1598b3dd376177fe1c38ed58b1477f"
+SIGKEYS["noxnox"]="daa74de3bf1aa87301a28ac9081d021de0c92299ec457d177014a026c888d288"
+SIGKEYS["tomh"]="bead63b9e44f5243e3030a37fdc0d1cd3efce65234c7bedfcff6ae6452d42e79"
+SIGKEYS["skorpy"]="0ebac3d341673dbeb8b6d2499811ce7851516aae851d71067a3e16488dee44c7"
+SIGKEYS["alex"]="5b8ce650fc50d845975567bd5418fcd5c091528e48e95cf0e2f0266ed509e013"
+SIGKEYS["build.ffda.io"]="24f20f0e0d7711181c70c85a76dda08334a96acd631994ace9b61b57a159db7b"
+SIGKEYS["github-actions-ci"]="cea1e84bf157d7362287fcd21d13de14634341e3d1ea7038000062743554dc88"
```

Comment:
Extract these from the site.conf, do not redefine.

Reply:
Do you have an idea how to extract them with reasonable effort and without hardcoding quite a few assumptions? It's not like a bit of lua code would suffice to read that file as we're also interested in the comment behind the key and not only the key itself?